### PR TITLE
updates for human pose estimation

### PR DESCRIPTION
**Pull request by zeyu-liu**
fix bugs and remove redundant code

### CODE REVIEW

File: fluid/PaddleCV/human_pose_estimation/README.md [hunk 1]
```diff
@@ -10,23 +10,28 @@ This is a simple demonstration of re-implementation in [PaddlePaddle.Fluid](http
...
+| 384x384\_pose\_resnet\_50 in PyTorch | 96.658 | 95.754 | 89.790 | 84.614 | 88.523 | 84.666 | 79.287 | 89.066 | 38.046 | - |
+| 384x384\_pose\_resnet\_50 in Fluid   | 96.862 | 95.635 | 90.046 | 85.557 | 88.818 | 84.948 | 78.484 | 89.235 | 38.093 | [`link`]() |
+
+## Results on COCO val2017 with detector having human AP of 56.4 on COCO val2017 dataset
+| Arch | AP | Ap .5 | AP .75 | AP (M) | AP (L) | AR | AR .5 | AR .75 | AR (M) | AR (L) | Models |
+| ---- |:--:|:-----:|:------:|:------:|:------:|:--:|:-----:|:------:|:------:|:------:|:------:|
+| 256x192\_pose\_resnet\_50 in PyTorch | 0.704 | 0.886 | 0.783 | 0.671 | 0.772 | 0.763 | 0.929 | 0.834 | 0.721 | 0.824 | - |
+| 256x192\_pose\_resnet\_50 in Fluid   | 0.712 | 0.897 | 0.786 | 0.683 | 0.756 | 0.741 | 0.906 | 0.806 | 0.709 | 0.790 | [`link`]() |
+| 384x288\_pose\_resnet\_50 in PyTorch | 0.722 | 0.893 | 0.789 | 0.681 | 0.797 | 0.776 | 0.932 | 0.838 | 0.728 | 0.846 | - |
+| 384x288\_pose\_resnet\_50 in Fluid   | 0.727 | 0.897 | 0.796 | 0.690 | 0.783 | 0.754 | 0.907 | 0.813 | 0.714 | 0.814 | [`link`]() |
```
Comment (qingqing01) [inline fluid/PaddleCV/human_pose_estimation/README.md]: Need to update link and doc in  part https://github.com/zeyu-liu/models/tree/develop/fluid/PaddleCV/human_pose_estimation#perform-validating

File: fluid/PaddleCV/human_pose_estimation/utils/mpii_evaluator.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2018-present, Baidu, Inc.
```
Comment (qingqing01) [inline fluid/PaddleCV/human_pose_estimation/utils/mpii_evaluator.py]: 2018 -> 2019

same as above

File: fluid/PaddleCV/human_pose_estimation/README.md [hunk 3]
```diff
@@ -10,23 +10,28 @@ This is a simple demonstration of re-implementation in [PaddlePaddle.Fluid](http
...
 
 ## Results on MPII Val
 | Arch | Head | Shoulder | Elbow | Wrist | Hip | Knee | Ankle | Mean | Mean@0.1| Models |
 | ---- |:----:|:--------:|:-----:|:-----:|:---:|:----:|:-----:|:----:|:-------:|:------:|
-| 383x384\_pose\_resnet\_50 in PyTorch | 96.658 | 95.754 | 89.790 | 84.614 | 88.523 | 84.666 | 79.287 | 89.066 | 38.046 | - |
-| 383x384\_pose\_resnet\_50 in Fluid   | 96.248 | 95.346 | 89.807 | 84.873 | 88.298 | 83.679 | 78.649 | 88.767 | 37.374 | [`link`](http://paddlemodels.bj.bcebos.com/pose/pose-resnet-50-384x384-mpii.tar.gz) |
+| 256x256\_pose\_resnet\_50 in PyTorch | 96.351	| 95.329 | 88.989 | 83.176 | 88.420	| 83.960 | 79.594 | 88.532 | 33.911 | - |
+| 256x256\_pose\_resnet\_50 in Fluid   | 96.385 | 95.363 | 89.211 | 84.084 | 88.454 | 84.182 | 79.546 | 88.748 | 33.750 | [`link`]() |
+| 384x384\_pose\_resnet\_50 in PyTorch | 96.658 | 95.754 | 89.790 | 84.614 | 88.523 | 84.666 | 79.287 | 89.066 | 38.046 | - |
+| 384x384\_pose\_resnet\_50 in Fluid   | 96.862 | 95.635 | 90.046 | 85.557 | 88.818 | 84.948 | 78.484 | 89.235 | 38.093 | [`link`]() |
```
Comment (qingqing01) [inline fluid/PaddleCV/human_pose_estimation/README.md]: Need to tell users the hype-parameters to reproduce the model

File: fluid/PaddleCV/human_pose_estimation/train.py [hunk 4]
```diff
@@ -30,18 +30,18 @@
...
-add_arg('num_epochs',       int,   140,                  "Number of epochs.")
-add_arg('total_images',     int,   144406,               "Training image number.")
-add_arg('kp_dim',           int,   16,                   "Class number.")
-add_arg('model_save_dir',   str,   "output",             "Model save directory")
-add_arg('with_mem_opt',     bool,  True,                 "Whether to use memory optimization or not.")
-add_arg('pretrained_model', str,   None,                 "Whether to use pretrained model.")
-add_arg('checkpoint',       str,   None,                 "Whether to resume checkpoint.")
-add_arg('lr',               float, 0.001,                "Set learning rate.")
-add_arg('lr_strategy',      str,   "piecewise_decay",    "Set the learning rate decay strategy.")
+add_arg('batch_size',       int,   128,                          "Minibatch size.")
```
Comment (qingqing01) [inline fluid/PaddleCV/human_pose_estimation/train.py]: Minibatch size per GPU? or total? need more clearly

File: fluid/PaddleCV/human_pose_estimation/train.py [hunk 5]
```diff
@@ -30,18 +30,18 @@
...
+add_arg('dataset',          str,   'mpii',                       "Dataset")
```
Comment (qingqing01) [inline fluid/PaddleCV/human_pose_estimation/train.py]: Give the all candidate lists

File: fluid/PaddleCV/human_pose_estimation/utils/base_evaluator.py [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+        """
+        :param root: the root dir of dataset
+        :param kp_dim: the dimension of keypoints
+        """
+        self.root = root
+        self.kp_dim = kp_dim
+
+    def evaluate(self, *args, **kwargs):
+        """
+        TODO: Implementation for specific task / dataset
```
Comment (qingqing01) [inline fluid/PaddleCV/human_pose_estimation/utils/base_evaluator.py]: Need TODO  here ?

File: fluid/PaddleCV/human_pose_estimation/README.md (hunk 1 above)
Comment (zeyu-liu) [reply fluid/PaddleCV/human_pose_estimation/README.md]: Done

File: fluid/PaddleCV/human_pose_estimation/utils/mpii_evaluator.py (hunk 2 above)
Comment (zeyu-liu) [reply fluid/PaddleCV/human_pose_estimation/utils/mpii_evaluator.py]: Done

File: fluid/PaddleCV/human_pose_estimation/README.md (hunk 3 above)
Comment (zeyu-liu) [reply fluid/PaddleCV/human_pose_estimation/README.md]: Done

File: fluid/PaddleCV/human_pose_estimation/train.py (hunk 4 above)
Comment (zeyu-liu) [reply fluid/PaddleCV/human_pose_estimation/train.py]: Done

File: fluid/PaddleCV/human_pose_estimation/train.py (hunk 5 above)
Comment (zeyu-liu) [reply fluid/PaddleCV/human_pose_estimation/train.py]: Done

File: fluid/PaddleCV/human_pose_estimation/utils/base_evaluator.py (hunk 6 above)
Comment (zeyu-liu) [reply fluid/PaddleCV/human_pose_estimation/utils/base_evaluator.py]: No need, will be removed in next commit.

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**